cmd/xgql/main.go

```diff
@@ -203,6 +203,7 @@ func main() {
 		clients.WithRESTMapper(rm),
 		clients.DoNotCache(noCache),
 		clients.WithLogger(log),
+		clients.WithExpiry(14*24*time.Hour), // Set the cache expire at 2 weeks for now
```

Comment:
@thephred could we pull this out to a config value that could be set via helm?

Reply:
Good idea 👍